test/dummy/db/seeds.rb

```diff
@@ -9,21 +9,23 @@ def clean_database
 end
 
 class JobsLoader
-  attr_reader :application, :server, :failed_jobs_count, :regular_jobs_count, :finished_jobs_count
+  attr_reader :application, :server, :failed_jobs_count, :regular_jobs_count, :finished_jobs_count, :blocked_jobs_count
 
   def initialize(application, server, failed_jobs_count: 100, regular_jobs_count: 50)
     @application = application
     @server = server
     @failed_jobs_count = randomize(failed_jobs_count)
     @regular_jobs_count = randomize(regular_jobs_count)
     @finished_jobs_count = randomize(regular_jobs_count)
+    @blocked_jobs_count = randomize(regular_jobs_count)
   end
 
   def load
     server.activating do
       load_finished_jobs
       load_failed_jobs
       load_regular_jobs
+      load_blocked_jobs if server.name == "solid_queue"
```

Comment:
I'd rely here on the adapter's exposed supported statuses. That's it, `if server.queue_adapter.supported_statuses.include?(:blocked)` or something like that.